sass/_SimpleFilter.scss

```diff
@@ -166,8 +169,22 @@
   margin: 0 5px;
   @include justify-content(flex-end);
   color: $color-greyish-teal-blue;
-  width: 10px;
-  height: 6px;
+  width: 12px;
+  height: 8px;
+}
+
+.coveo-simplefilter-eraser {
+  position: relative;
+  top: 1px;
+  margin: 0 5px 0 10px;
+  cursor: pointer;
```

Comment:
I think this is already included in the `clearButton` mixin

Reply:
Nope. There aren't . I took those properties from `.coveo-facet-header-eraser` inside `_FacetHeader.scss`.  Maybe we could create something like a `baseHeaderButton` mixin to reuse more properties.